src/west/commands/config.py

```diff
@@ -200,8 +200,8 @@ def write(self, args):
     def _sk(self, args):
         name_list = args.name.split(".", 1)
         if len(name_list) != 2:
-            self.parser.error('name {} should be in the form <section>.<key>',
-                              exit_code=3)
+            self.parser.error("name '{}' should be in the form <section>.<key>".
```

Comment:
Actually, can you please fix the shippable error and keep this line 80 column clean?

Reply:
Should be good now.